applications/ContactStructuralMechanicsApplication/custom_processes/find_intersected_geometrical_objects_with_obb_for_contact_search_process.cpp

```diff
@@ -23,7 +23,7 @@ FindIntersectedGeometricalObjectsWithOBBContactSearchProcess::FindIntersectedGeo
     ModelPart& rPart1,
     ModelPart& rPart2,
     const double BoundingBoxFactor,
-    const Flags Options
+    const Flags& Options
```

Comment:
Is it necessary to pass `Kratos::Flags` by reference?

@pooyan-dadvand I thought this was a super lightweight class?

Reply:
I am quite sure Kratos::Flags > double. So a priori is better. In addition, making a copy maybe not be much more in memory, but there is a computation cost